pdg/src/info.rs

```diff
@@ -308,12 +316,7 @@ mod test {
         let b3 = mk_store_addr(&mut g, b1);
 
         let pdg = build_pdg(g);
-        assert!(!info(&pdg, a).unique);
-        assert!(!info(&pdg, b1).unique);
-        assert!(!info(&pdg, b2).unique);
-        assert!(!info(&pdg, b3).unique);
-        assert!(!info(&pdg, c1).unique);
-        assert!(!info(&pdg, c2).unique);
+        check_unique(&pdg, &[], &[a, b1, b2, b3, c1, c2]);
```

Comment:
having these two arguments really doesn't make clear that you're checking for non-uniqueness here, in addition to the function name. i suggest having one function for each

Reply:
Yeah, that might be better.  Might not be worth changing, though, given #668.